pkg/agent/datamodel/types.go

```diff
@@ -122,6 +122,14 @@ const (
 	WasmWasi WorkloadRuntime = "WasmWasi"
 )
 
+// OutboundType describes the options for outbound internet access.
+type OutboundType int
+
+const (
+	OutboundTypeNone OutboundType = iota
```

Comment:
if you wanna use the enum then start with"

`OutboundTypeUnspecified = iota` instead

Reply:
(since this will have value 0 and the zero-value of int is also 0)